docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx

```diff
@@ -0,0 +1,345 @@
+2. In the search box, type **IBM LMIC framework** into the search, select the version shown below, and click Install.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruit-feather-m0-rfm95-arduino-add-library.png"
+  )}
+/>
+
+To communicate with the DHT22 sensor, we'll need to install an Arduino library.
+
+To install, open your Arduino IDE:
+
+1. Navigate to Library Manager \(**Sketch &gt; Include Library &gt; Manage Libraries**\).
+2. In the search box, type **DHT22 sensor library** into the search, select the version shown below, and click Install.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruit-feather-m0-rfm95-arduino-add-dht22-library.png"
+  )}
+/>
+
+To encode our data with CayenneLPP, we'll need to install an Arduino library.
+
+To install, open your Arduino IDE:
+
+1. Navigate to Library Manager \(**Sketch &gt; Include Library &gt; Manage Libraries**\).
+2. In the search box, type **CayenneLPP** into the search, select the version shown below, and click Install.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruit-feather-m0-rfm95-arduino-add-cayennelpp-library.png"
+  )}
+/>
+
+#### Update IBM LMIC framework config.h
+
+This library requires that a config file be setup properly. After you have installed the `IBM LMIC framework` library, navigate to it's directory on your operating system found below. Next, replace the `config.h` file in this directory with this [config.h](https://github.com/helium/longfi-arduino/blob/master/Adafruit-Feather-M0-RFM95/longfi-us915/config.h) file.
+
+linux: /home/{user}/Arduino/libraries/IBM\_LMIC\_framework/src/lmic  
+windows: Documents/Arduino/libraries/IBM\_LMIC\_framework/src/lmic  
+mac os: Documents/Arduino/libraries/IBM\_LMIC\_framework/src/lmic
+
+### Programming Example Sketch
+
+Now that we have the required Arduino board support and library installed, lets program the board with the provided example sketch.
+
+To create a new Arduino sketch, open your Arduino IDE, \(**File &gt; New\).** Next, replace the template sketch with the sketch found [here](https://github.com/helium/longfi-arduino/blob/master/Adafruit-Feather-M0-RFM95/longfi-us915-cayenne-dht22/longfi-us915-cayenne-dht22.ino), copy and paste the entirety of it.
+
+Next we'll need to fill in the AppEUI\(lsb\), DevEUI\(lsb\), and AppKey\(msb\), in the sketch, which you can find on the device details page on Console. Be sure to use the formatting buttons to match the endianess and formatting required for the sketch, shown below.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruit-feather-m0-rfm95-console-keys.png"
+  )}
+/>
+
+At the top of the sketch, replace the three **FILL\_ME\_IN** fields, with the matching field from Console, example shown below.
```

Comment:
Would it be better to have an embedded code sample here people can copy and paste?

Reply:
It's much too long unfortunately and we would need to update the docs if any changes are made to the code. Perhaps once we find a good way of pulling it from github and collapsing the view.

Reply:
Roger. Screenshot works.